agent/config/config_unix.go

```diff
@@ -49,6 +49,8 @@ func DefaultConfig() Config {
 		DisableMetrics:              false,
 		ReservedMemory:              0,
 		AvailableLoggingDrivers:     []dockerclient.LoggingDriver{dockerclient.JSONFileDriver, dockerclient.NoneDriver},
+		PollMetrics:                 false,
```

Comment:
Can you modify the test in `config_test.go` to cover this?  Specifically, the following test:
https://github.com/aws/amazon-ecs-agent/blob/331f018333c70fe408f20028a38a3a28b03636dc/agent/config/config_test.go#L68

and add a test like this to test the default behavior:
https://github.com/aws/amazon-ecs-agent/blob/331f018333c70fe408f20028a38a3a28b03636dc/agent/config/config_test.go#L184

Reply:
Yes. Updated. I added tests to for testing the max and min poll interval also.